modules/ding_place2book/ding_place2book.module

```diff
@@ -679,6 +665,31 @@ function ding_place2book_get_library_nid($entity) {
   return $library_id;
 }
 
+
+/**
+ * Gets the place2book id for a node.
+ *
+ * @param object $node
+ *   The node to check.
+ *
+ * @return int|NULL
+ *   The corresponding place2book id for the node or NULL if the node does not
+ *   have a place2book id.
+ */
+function ding_place2book_get_id($node) {
```

Comment:
Great to have a generic function 👍 I think this should be used in `ding_place2book_node_delete()` as well.